tests/rules/test_rails_migrations_pending.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from thefuck.rules.rails_migrations_pending import match, get_new_command
+from thefuck.types import Command
+
+output_env_development = '''
+Migrations are pending. To resolve this issue, run:
+
+        rails db:migrate RAILS_ENV=development
+'''
+output_env_test = '''
+Migrations are pending. To resolve this issue, run:
+
+        bin/rails db:migrate RAILS_ENV=test
+'''
+
+
+@pytest.mark.parametrize(
+    "command",
+    [
+        Command("", output_env_development),
+        Command("", output_env_test),
+    ],
+)
+def test_match(command):
+    assert match(command)
+
+
+@pytest.mark.parametrize(
+    "command",
+    [
+        Command("Environment data not found in the schema. To resolve this issue, run: \n\n", ""),
+    ],
+)
+def test_not_match(command):
+    assert not match(command)
+
+
+@pytest.mark.parametrize(
+    "command, new_command",
+    [
+        (Command("", output_env_development), "rails db:migrate RAILS_ENV=development"),
+        (Command("", output_env_test), "bin/rails db:migrate RAILS_ENV=test"),
```

Comment:
Reviewing it again, I thought that the corrected command could include the one initially intended by the user.

```suggestion
        (Command("bin/rspec", output_env_development), "rails db:migrate RAILS_ENV=development && bin/rspec"),
        (Command("bin/rspec", output_env_test), "bin/rails db:migrate RAILS_ENV=test && bin/rspec"),
```

Make sure to use `shell._and` to join the commands together.

Reply:
Thanks, done